[2024-12-05T11:43:09.778+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning manual__2024-12-05T11:43:06.806334+00:00 [queued]>
[2024-12-05T11:43:09.786+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.data-cleaning manual__2024-12-05T11:43:06.806334+00:00 [queued]>
[2024-12-05T11:43:09.786+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T11:43:09.786+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-12-05T11:43:09.787+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T11:43:09.799+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): data-cleaning> on 2024-12-05 11:43:06.806334+00:00
[2024-12-05T11:43:09.805+0000] {standard_task_runner.py:52} INFO - Started process 546 to run task
[2024-12-05T11:43:09.807+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'P2M3', 'data-cleaning', 'manual__2024-12-05T11:43:06.806334+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_alridho_DAG.py', '--cfg-path', '/tmp/tmpg3x5657k', '--error-file', '/tmp/tmp7mqlar0c']
[2024-12-05T11:43:09.808+0000] {standard_task_runner.py:80} INFO - Job 36: Subtask data-cleaning
[2024-12-05T11:43:09.903+0000] {task_command.py:371} INFO - Running <TaskInstance: P2M3.data-cleaning manual__2024-12-05T11:43:06.806334+00:00 [running]> on host 9b483ed80620
[2024-12-05T11:43:10.002+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alridho
AIRFLOW_CTX_DAG_ID=P2M3
AIRFLOW_CTX_TASK_ID=data-cleaning
AIRFLOW_CTX_EXECUTION_DATE=2024-12-05T11:43:06.806334+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-05T11:43:06.806334+00:00
[2024-12-05T11:43:10.032+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-12-05T11:43:10.055+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=P2M3, task_id=data-cleaning, execution_date=20241205T114306, start_date=20241205T114309, end_date=20241205T114310
[2024-12-05T11:43:10.099+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05T11:43:10.155+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
